### ISSUE TITLE
Official Godot 3.0+ stopped working and doesn't run anymore.

### ISSUE THREAD

**Issue by freemanfromgodotengine** [godotengine/godot#17480]
**Godot version:**
godot-3.0.2-official & godot-3.0-official

**OS/device including version:**
Manjaro XFCE Edition (17.1.6) Linux

**Issue description:**
I downloaded Godot. I run it, it RUN fine. Next I installed templates from the downloaded file and changed theme to Alien. I checked around what's going on in new version, were surprised not to see hidden /.godot directory in my users home folder etc., and not able to find my way back to project manager, I decided to closed Godot and run again. It closed all right.
Now when I'm trying to run Godot again, I am only able to see an opening window's frames for a moment, and I see in terminal:

When I try to run 3.0.2-stable:
> OpenGL ES 3.0 Renderer: AMD JUNIPER (DRM 2.50.0 / 4.14.24-1-MANJARO, LLVM 5.0.1)
> Segmentation fault (core dumped)

When I try to run 3.0-stable:
> OpenGL ES 3.0 Renderer: AMD JUNIPER (DRM 2.50.0 / 4.14.24-1-MANJARO, LLVM 5.0.1)
> GLES3: max ubo light: 409
> GLES3: max ubo reflections: 455, ubo size: 144
> Segmentation fault (core dumped)

Errr... what's going on?

P.S. of course I restarted computer and tried again. :)
I also deleted godot, as well as folders
```
/home/me/.config/godot 
/home/me/.local/share/godot 
```
Nothing changed.

I encountered something very similar when I had a problem with the open source drivers on my nvidia laptop and Godot 2.1+, so here is my `glxinfo | grep 'version'` output:

> server glx version string: 1.4
> client glx version string: 1.4
> GLX version: 1.4
>     Max core profile version: 3.3
>     Max compat profile version: 3.0
>     Max GLES1 profile version: 1.1
>     Max GLES[23] profile version: 3.0
> OpenGL core profile version string: 3.3 (Core Profile) Mesa 17.3.6
> OpenGL core profile shading language version string: 3.30
> OpenGL version string: 3.0 Mesa 17.3.6
> OpenGL shading language version string: 1.30
> OpenGL ES profile version string: OpenGL ES 3.0 Mesa 17.3.6
> OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.00
> 

but like I mentioned earlier, for the first time it opened and run smoothly, so I doubt my Radeon HD 5770 has anything to do with that.

**Comment by akien-mga** [comment]
Would you be able to build from source in debug mode (default setting) to get a stacktrace from the crash?

**Comment by freemanfromgodotengine** [comment]
@akien-mga How would you like me to build it? Godot in debug is a new territory for me.
`scons platform=x11 debug_release=yes` or you need some additions like `warnings=` or `verbose=` etc?
Also, a little instruction what to do next would be helpful, as I did not use Godot in debug mode yet.

**Comment by Chaosus** [comment]
Last time when Godot completly stopped worked on my computer (im Linux user too), soon was the hard drive crash happens. I recommend you to backup important files as soon as possible..

**Comment by freemanfromgodotengine** [comment]
@Chaosus  Thanks for the tip. I will test the health of the hard drive, but disk seams to be working all right and actually is not that old to be failing. 
But of course backup is always a good idea.

**Comment by akien-mga** [comment]
I meant just `scons p=x11`, the resulting tools binary will habe debug symbols.
Then run it through gdb: `gdb bin/godot.x11.tools.64`, and Ehen it crashes getva backtrace with `bt full`.

**Comment by freemanfromgodotengine** [comment]
This is very peculiar, as it seams that @Chaosus was really into something.
Yesterday at the end of day I had to turn off my computer by long pressing power button, because of the total freeze that is always very unusual under Linux on my computers. And today when I was trying to turn my computer on, booting got stuck on the console message of "recovering journal", that did not work at all. I managed to run fsck while booting from different drive. After some investigation, I changed the usb cable and I was able to run the system again.
And guess what... Godot is starting just fine now!

Therfore I propose to additionally promote Godot Engine 3.0+ as an early warning disk/filesystem/usb cable problem detector :)

P.S. Thanks @akien-mga for the instruction. It will be helpful in the future.
